02.calendar/calendar.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'optparse'
+
+class Calender
+  def initialize(year: Date.today.year, month: Date.today.month)
```

Comment:
デフォルト値は呼び出し側で設定しているので、こっちにも書くと二重に定義しちゃうことになりますね。
なので、こっちはなくしちゃっていいかも？

```suggestion
  def initialize(year:, month:)
```